java-frontend/src/main/java/org/sonar/java/resolve/JavaSymbol.java

```diff
@@ -729,6 +729,11 @@ public MethodTree declaration() {
       return declaration;
     }
 
+    @Override
+    public boolean isOverridable() {
```

Comment:
This could be used in [`ConstructorCallingOverridableCheck`](https://github.com/SonarSource/sonar-java/blob/master/java-checks/src/main/java/org/sonar/java/checks/ConstructorCallingOverridableCheck.java#L135) as well.

There is also another implementation (testing it the other way), which is more complete, as checking for the `native` flag: https://github.com/SonarSource/sonar-java/blob/master/java-frontend/src/main/java/org/sonar/java/se/SymbolicExecutionVisitor.java#L94

You may want to have a look to it.